Baidu Scholar.js

```diff
@@ -106,16 +110,63 @@ function scrape(doc, url) {
 				item.abstractNote = text(doc, 'div.sc_abstract') || text(doc, 'p.abstract');
 			}
 			item.attachments.push({
-				title: "Snapshot",
-				document: doc
+				url,
+				title: "Baidu学术地址",
+				mimeType: "text/html",
+				snapshot: false,
 			});
+			// Sources
```

Comment:
I don't think sources and related do what you think they do -- they're not realy usable in web translators, so please delete this code.